src/main/java/ChatRequest.java

```diff
@@ -24,4 +24,47 @@ public boolean newMethod(String st1 ,String st2){
             }
             return false;
         }
+        public String chat(String question){
+            /* Initialize Variable */
+            PromptTemplate promptTemplate = null;
+            try {
+                /* Create a prompt template using the question */
+                promptTemplate = new PromptTemplate(question);
+            }catch(Exception e) {
+                log.error("error : "+e.getMessage());
```

Comment:
Logging the error message without the stack trace can make debugging difficult. Additionally, it may expose sensitive information depending on the context. (Severity: High)